docs/LTS.md

```diff
@@ -35,15 +35,14 @@ A "month" is to be a period of 30 consecutive days.
 | :------ | :----------- | :-------------- | :------------------ |
 | 1.0.0   | 2018-03-06   | 2019-09-01      | 6, 8, 9, 10, 11, 12 |
 | 2.0.0   | 2019-02-25   | TBD             | 6, 8, 10, 11, 12    |
+| 3.0.0   | TBD          | TBD             | 8, 10, 11, 12       |
 
 <a name="supported-os"></a>
 
 ### CI tested operating systems
 
 | CI              | OS      | Version        | Package Manager | Node.js               |
 | :-------------- | :------ | :------------- | :-------------- | :-------------------- |
-| Azure pipelines | Linux   | Ubuntu 16.04   | npm, yarn       | ~~6¹~~, 8, 10, 11, 12 |
-| Azure pipelines | Windows | vs2017-win2016 | npm, yarn       | ~~6¹~~, 8, 10, 11, 12 |
-| Azure pipelines | Mac     | macOS 10.14    | npm, yarn       | ~~6¹~~, 8, 10, 11, 12 |
-
-_¹ yarn supports only node >= 8_
+| Azure pipelines | Linux   | Ubuntu 16.04   | npm, yarn       | 8, 10, 11, 12 |
+| Azure pipelines | Windows | vs2017-win2016 | npm, yarn       | 8, 10, 11, 12 |
+| Azure pipelines | Mac     | macOS 10.14    | npm, yarn       | 8, 10, 11, 12 |
```

Comment:
I think node 11 support can be dropped as it EOL

Reply:
yes I agree ... @mcollina @delvedor is it ok even for you ?

Reply:
Yes, we can safely drop odd numbers (once the next even is out).

Reply:
done, thanks for clarifications